electronics_model/BomBackend.py

```diff
@@ -0,0 +1,48 @@
+import io
+from typing import List, Tuple, Dict, NamedTuple
+
+import edgir
+from edg_core import BaseBackend, CompiledDesign, TransformUtil
+
+import csv
+
+
+class BomItem(NamedTuple):
+    footprint: str
+    value: str
```

Comment:
Can you also add a field for JLC part number?

The column name in the CSV should be `JLCPCB Part #`, and the part parameter would be `lcsc_part`. Like value, this should be empty if None.

Reply:
done